src/modules/tests/tests.selector.ts

```diff
@@ -62,11 +65,14 @@ export const isPassed = (test: TestResultSchemasUnion): boolean => {
 };
 
 export const getActivityCode = (test: TestResultCommonSchema) => {
-
-  const activityCodeIndex = activityCodeModelList.findIndex(
-    activityCode => test.activityCode === activityCode.activityCode);
-  return activityCodeModelList[activityCodeIndex];
-
+  if (test.delegatedTest) {
```

Comment:
Can we verify no error for tests that don't have the 'delegatedTest' key such as in MOD1/MOD2

Reply:
tested with every cat, no errors.